modules/swagger-codegen/src/main/resources/Kotlin/build.gradle.kts.mustache

```diff
@@ -0,0 +1,109 @@
+import org.gradle.api.publish.maven.*
+
+import org.jfrog.gradle.plugin.artifactory.dsl.PublisherConfig
```

Comment:
Remove artifactory this from the default template here and put it in an internal template.

Reply:
Done, have removed it in platform-client-sdk-common also